src/main/java/redstone/multimeter/client/gui/Texture.java

```diff
@@ -4,7 +4,7 @@
 
 public class Texture {
 
-	public static final Texture OPTIONS_BACKGROUND    = new Texture(new ResourceLocation("textures/gui/options_background.png"), 16, 16);
+	public static final Texture OPTIONS_BACKGROUND    = new Texture(new ResourceLocation("textures/gui/inworld_menu_list_background.png"), 16, 16);
```

Comment:
IMO `menu_background` should be used instead, so as to keep the same aesthetic as previous versions. As you said in the issue report, this solution results in otherwise hidden elements being exposed, and I wouldn't accept it without that being fixed too.

Reply:
since this was a change made to all of the game's backgrounds, keeping the old aesthetic would make it inconsistent with the game, which i think would feel more out of place.
`menu_background` is also a translucent texture and results in the same issue
i chose `inworld_menu_list_background` specifically because there is a keybind for opening the config while in-game (i didn't realize you can also open it from the main menu, so maybe the non-in-game variant would be better)

Reply:
I poked around a little and it seems that keeping the old aesthetic would be more of a pain than this, so I'll just go along with it